app/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -13,12 +13,21 @@ import { NotificationService } from '../notification.service';
 })
 export class NotificationsListComponent {
   @Input() notifications: Notification[];
+  @Input() selectedStatus: string = '';
```

Comment:
All three added variables are not really needed. One can apply the pipe directly in the  `notifications-list.component.html` . Then nothing has to be changed in the notifications list - neither the `notifications-list.component.ts` nor the `notifications-list.component.html`.